src/test/fixtures/formularios.yml

```
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  data_abertura: 2025-07-06 16:29:53
+  datra_fechamento: 2025-07-06 16:29:53
+
+two:
+  data_abertura: 2025-07-06 16:29:53
+  datra_fechamento: 2025-07-06 16:29:53
```

Erro de escrita em "datRa_fechamento" acusado por [teste](https://github.com/diogob003/CAMAAR/actions/runs/16146384648/job/45566228008?pr=17).
```suggestion
  data_fechamento: 2025-07-06 16:29:53

two:
  data_abertura: 2025-07-06 16:29:53
  data_fechamento: 2025-07-06 16:29:53
```